ts_scripts/print_env_info.py

```diff
@@ -50,8 +50,24 @@
     "cudnn_version": []
 }
 
-smi = 'nvidia-smi'
+npm_env =  {
+    "npm_pkg_version": []
+}
 
+def get_nvidia_smi():
+    # Note: nvidia-smi is currently available only on Windows and Linux
+    smi = 'nvidia-smi'
+    if get_platform() == 'win32':
```

Comment:
Windows has `nvidia-smi.exe` instead of `nvidia-smi`. We should only check for that.

See if we can reuse this dictionary from [ts_scripts/util.py](https://github.com/pytorch/serve/blob/master/ts_scripts/utils.py#L5)

Reply:
updated